website/pages/en/index.js

```diff
@@ -84,54 +74,247 @@ class Index extends React.Component {
       <div>
         <HomeSplash language={language} />
         <div className="mainContainer">
-          <Container padding={['bottom', 'top']}>
+          <Container className="introSection" background="light">
             <GridBlock
-              align="center"
               contents={[
                 {
                   content:
-                    'Declare the data your components need with GraphQL, Relay determines how and when to fetch your data.',
-                  image: '',
-                  imageAlign: 'top',
-                  title: 'Declarative',
+                    "<p>Relay provides a structured way to build data-driven React applications safely.</p> <p>Relay does this by letting each React component declare the parts of a GraphQL schema it needs, and then Relay handles merging these fragments into single query and passing the response data to those components.</p><p>This allows you to have a declarative API for your app's data</p>",
                 },
+              ]}
+            />
+          </Container>
+
+          <Container className="exampleSection">
+            <div className="wrapperInner">
+              <div className="radiusRight">
+                <h2>Query Renderer</h2>
+                <p>
+                  When creating a new screen, you start with a <a href="/docs/en/query-renderer"><code>QueryRenderer</code></a>. 
+                </p>
+                <p>
+                  A <code>QueryRenderer</code> is a React Component which is the root of a Relay component tree. It handles fetching your query, and uses the <code>render</code> prop to render the resulting data.
+                </p>
+                <p>
+                  As React components, <code>QueryRenderers</code> can be rendered anywhere that a React component can be rendered, i.e. not just at the top level but *within* other components or containers. For example, you could use a <code>QueryRenderer</code> to lazily fetch additional data for a popover.
+                </p>
+              </div>
+
+              <div className="radiusLeft">
+              
+                <pre>
+                  <code>
+                  {`
+import React from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import environment from "./lib/createRelayEnvironment"
+import ArtistHeader from “./ArtistHeader” // Below
+
+// You can usually have use one query renderer per page
+// and it represents the root of a query
+export const ArtistRenderer = ({ artistID }) => {
+  return (
+    <QueryRenderer
+      environment={environment}
+      query={graphql\`
+        query QueryRenderersArtistQuery($artistID: String!) {
```

Comment:
Are you sure this will work? Afaik queries need to end in `_Query`?

Reply:
The underscore is not necessary.